### PR TITLE
[MFTF] Use action group go to ProductCatalogPage

### PR DESCRIPTION
**Pull request by Usik2203**
This PR uses `AdminProductCatalogPageOpenActionGroup` go to ProductCatalogPage instead 
`<amOnPage url="{{ProductCatalogPage.url}}" stepKey="GoToCatalogPageChangingView"/>`
        `<waitForPageLoad stepKey="WaitForPageToLoadFullyChangingView"/>`

### CODE REVIEW

**Comment by Usik2203** [comment]
@magento run Functional Tests CE , Functional Tests EE , Functional Tests B2B

File: app/code/Magento/Bundle/Test/Mftf/Test/MassEnableDisableBundleProductsTest.xml [hunk 1]
```diff
@@ -130,8 +130,7 @@
         <dontSeeElement stepKey="LookingForNameOfProductDisabled" selector="{{StorefrontBundledSection.bundleProductName}}"/>
 
         <!--Enabling bundle products-->
-        <amOnPage url="{{ProductCatalogPage.url}}" stepKey="GoToCatalogPageChangingView"/>
-        <waitForPageLoad stepKey="WaitForPageToLoadFullyChangingView"/>
+        <actionGroup ref="AdminProductCatalogPageOpenActionGroup" stepKey="GoToCatalogPageChangingView"/>
```
Comment (eduard13) [inline app/code/Magento/Bundle/Test/Mftf/Test/MassEnableDisableBundleProductsTest.xml]: Lets following the camelcase format, the 1st letter should be a lowercase.
```suggestion
        <actionGroup ref="AdminProductCatalogPageOpenActionGroup" stepKey="goToCatalogPageChangingView"/>
```

File: app/code/Magento/Catalog/Test/Mftf/ActionGroup/AdminProductCatalogPageOpenActionGroup.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminProductCatalogPageOpenActionGroup">
+        <annotations>
+            <description>Goes to the Admin Product Catalog Page grid page.</description>
+        </annotations>
+
+        <amOnPage url="{{ProductCatalogPage.url}}" stepKey="OpenProductCatalogPage"/>
```
Comment (eduard13) [inline app/code/Magento/Catalog/Test/Mftf/ActionGroup/AdminProductCatalogPageOpenActionGroup.xml]:
```suggestion
        <amOnPage url="{{ProductCatalogPage.url}}" stepKey="openProductCatalogPage"/>
```

**Review by eduard13 — CHANGES_REQUESTED** [review]
Good catch, however lets follow the camelCase naming convention here and use a lower case for the 1st letter. Please implement the same changes to all the places in this PR.
Thank you.

**Comment by Usik2203** [comment]
@magento run Functional Tests CE , Functional Tests EE , Functional Tests B2B

**Comment by Usik2203** [comment]
@magento run Functional Tests B2B

File: app/code/Magento/Catalog/Test/Mftf/ActionGroup/AdminProductCatalogPageOpenActionGroup.xml (hunk 2 above)
Comment (Usik2203) [reply app/code/Magento/Catalog/Test/Mftf/ActionGroup/AdminProductCatalogPageOpenActionGroup.xml]: **Done**

File: app/code/Magento/Bundle/Test/Mftf/Test/MassEnableDisableBundleProductsTest.xml (hunk 1 above)
Comment (Usik2203) [reply app/code/Magento/Bundle/Test/Mftf/Test/MassEnableDisableBundleProductsTest.xml]: **Done**

**Review by eduard13 — APPROVED** [review]
Awesome 👍

**Comment by eduard13** [comment]
@magento run all tests

**Comment by Usik2203** [comment]
@magento run Adobe CLA Signed

**Comment by eduard13** [comment]
The failing tests don't seem to be related to the PR changes.

**Review by eduard13 — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @eduard13, thank you for the review.
[ENGCOM-7783](https://jira.corp.magento.com/browse/ENGCOM-7783) has been created to process this Pull Request

**Comment by engcom-Alfa** [comment]
QA not applicable

**Pull request merged**